app/src/androidTest/java/com/github/multimatum_team/multimatum/MainActivityTest.kt

```diff
@@ -237,5 +240,14 @@ class MainActivityTest {
                     Deadline("Test 3", DeadlineState.TODO, LocalDateTime.of(2022, 3, 7, 0, 0))
                 )
             )
+
+        @Singleton
+        @Provides
+        fun provideAuthRepository(): AuthRepository =
+            MockAuthRepository()
+    }
+
+    companion object {
```

Comment:
I'm not really sure. Why this need to be in a companion object ?

Reply:
Before I moved this test to the roboelectric tests, the mockSharedPreferences was taken from the MainSettingsActivityTest, which I kept in the instrumented tests. Therefore I simply copied what was in MainSettingsActivityTest and put it in MainActivityTest.